### PR TITLE
fix warpctc plugin work with new version warpctc

### PR DESCRIPTION
**Pull request by yajiedesign**

### CODE REVIEW

**Comment by piiswrong** [comment]
Are you sure this fix works? In the issue someone said after the fix gpu inference still doesn't work

**Comment by yajiedesign** [comment]
i am test it work .
The key point is `info.blank_label` = 0;`

File: plugin/warpctc/warpctc-inl.h [hunk 1]
```diff
@@ -120,7 +120,7 @@ class WarpCTCOp : public Operator {
     TBlob data = in_data[warpctc_enum::kData];
     TBlob label = in_data[warpctc_enum::kLabel];
     CHECK_EQ(data.shape_.ndim(), 2) << "input data shape should be 2 (t*n, p)";
-    ctcComputeInfo info; // please build warp-ctc with commit 5bfb46e (cd warp-ctc && git checkout 5bfb46e) NOLINT(*)
+    ctcOptions info;
```
Comment (piiswrong) [inline plugin/warpctc/warpctc-inl.h]: cool. add a comment in this line: please updated to latest baidu/warp-ctc

Comment (yajiedesign) [inline plugin/warpctc/warpctc-inl.h]: ok

**Comment by piiswrong** [comment]
windows build is failing @yajiedesign

**Pull request merged**